<?xml version="1.0"?>

<launch>
  <!--PARAMETERS-->
  <param name="rawlog_play_delay" value="0.01"/>
  <param name="ini_filename" value="$(find mrpt_icp_slam_2d)/tutorial/icp_slam_demo.ini"/>

  <param name="rawlog_filename" value="$(find mrpt_icp_slam_2d)/tutorial/icp_slam_demo.rawlog"/>
  <param name="odom_frame_id" value="/odom"/>
  <param name="global_frame_id" value="/map"/>
  <param name="base_frame_id" value="/base_link"/>
  <!-- for multiple sensors, place comma eg "r1/front_laser/scan,/scan2" -->
  <param name="sensor_source" value="/scan"/>
  <param name="trajectory_update_rate" value="10"/>
  <param name="trajectory_publish_rate" value="5"/>

  <env name="ROSCONSOLE_CONFIG_FILE" value="$(find mrpt_icp_slam_2d)/config/rosconsole.config"/>

  <!--SLAM RUN-->
  <node pkg="rviz" type="rviz" name="rviz_nav" output="screen" args="-d $(find mrpt_icp_slam_2d)/rviz/rviz_conf.rviz"></node>
  <node pkg="mrpt_icp_slam_2d" type="mrpt_icp_slam_2d" name="mrpt_icp_slam_2d" output="screen"></node>

</launch>
